cli/src/execDocker/docker.js

```diff
@@ -3,6 +3,13 @@ import os from 'os';
 
 const docker = new Docker();
 
+const abortController = new AbortController();
+const { signal } = abortController;
+
+process.on('SIGINT', () => {
+  abortController.abort();
+});
```

Comment:
Registering the listener in the module scope and never cleaning it is problematic because it prevents SIGINT from stopping the process in the whole app.

This creates this bug:
![image](https://github.com/user-attachments/assets/9557dffe-a2d8-4bfb-90a0-efa3e66df1d8)
I cannot cancel the command while a long async step is running (in the example TEE transformation step)

We need to register the SIGINT listener just before calling docker and clean it as soon as the docker interaction ends (successfully or with an error).